python/pyspark/sql/pandas/utils.py

```diff
@@ -26,11 +26,12 @@ def require_minimum_pandas_version():
     try:
         import pandas
         have_pandas = True
-    except ImportError:
+    except ImportError as error:
         have_pandas = False
+        raised_error = error
```

Comment:
I would just remove this one though.

Reply:
Addressed in b21b67a187b59d5dae6e2eb7e93e53f06574a5c4 :)

Reply:
Looks like we can't do that https://github.com/apache/spark/pull/31902/checks?check_run_id=2166094118